datasets/swedish_medical_ner/README.md

```diff
@@ -0,0 +1,192 @@
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Repository:** https://github.com/olofmogren/biomedical-ner-data-swedish
+- **Paper:** [Named Entity Recognition in Swedish Health Records with Character-Based Deep Bidirectional LSTMs](https://aclanthology.org/W16-5104.pdf)
+- **Point of Contact:** [Olof Mogren](olof@mogren.one)
+
+### Dataset Summary
+
+SwedMedNER is Named Entity Recognition dataset on medical text in Swedish. It consists three subsets which are in turn derived from three different sources respectively: the Swedish Wikipedia (a.k.a. wiki), Läkartidningen (a.k.a. lt), and 1177 Vårdguiden (a.k.a. 1177). While the Swedish Wikipedia and Läkartidningen subsets in total contains over 790000 sequences with 60 characters each, the 1177 Vårdguiden subset is manually annotated and contains 927 sentences, 2740 annotations, out of which 1574 are _disorder and findings_, 546 are _pharmaceutical drug_, and 620 are _body structure_.
+
+Texts from both Swedish Wikipedia and Läkartidningen were automatically annotated using a list of medical seed terms. Sentences from 1177 Vårdguiden were manuually annotated.
+
+
+### Supported Tasks and Leaderboards
+
+Medical NER.
+
+### Languages
+
+Swedish (SV).
+
+## Dataset Structure
+
+### Data Instances
+
+Annotated example sentences are shown below:
+
+```
+( Förstoppning ) är ett vanligt problem hos äldre.
+[ Cox-hämmare ] finns även som gel och sprej.
+[ Medicinen ] kan också göra att man blöder lättare eftersom den påverkar { blodets } förmåga att levra sig.
+```
+
+Tags are as follows:
+- Prenthesis, (): Disorder and Finding
+- Brackets, []: Pharmaceutical Drug
+- Curly brackets, {}: Body Structure
+
+Data example:
+
+```
+In: data = load_dataset('./datasets/swedish_medical_ner', "wiki")
+In: data['train']:
+Out: 
+Dataset({
+    features: ['sid', 'sentence', 'entities'],
+    num_rows: 48720
+})
+
+In: data['train'][0]['sentence']
+Out: '{kropp} beskrivs i till exempel människokroppen, anatomi och f'
```

Comment:
I will leave this to the end users. I am not sure if I should edit the original data much given the data licence. As of now the index fields match with the brackets in the sentences.

Reply:
Ok since the index match the text with the brackets it's fine.
Note that the data license shouldn't be an issue if we want to add further data processing in the script